velox/vector/tests/utils/VectorTestBase.h

```diff
@@ -459,6 +459,21 @@ class VectorTestBase {
         size, sizeAt, valueAt, isNullAt, valueIsNullAt, arrayType);
   }
 
+  /// Similar to makeArrayVectorFromJson, Creates an ArrayVector from list of
+  /// JSON arrays of arrays.
+  /// @tparam T Type of array elements. Must be an integer: int8_t, int16_t,
+  /// int32_t, int64_t.
+  /// @param jsonArrays A list of JSON arrays. JSON array cannot be an empty
+  /// string.
+  /// @param arrayType type of array elements as defined by user. Defaults
```

Comment:
> Defaults to T type if not specified

Is this correct?

Make sure to end all sentences with a period.

Reply:
was just trying to mention that the variable defaults to  ARRAY(CppToType<T>::create();

perhaps the correct wording is: "Defaults to to Array of T type if not specified."